go/protocol/keybase1/kex2provisionee2.go

```diff
@@ -9,8 +9,15 @@ import (
 )
 
 type Hello2Res struct {
-	EncryptionKey KID      `codec:"encryptionKey" json:"encryptionKey"`
-	SigPayload    HelloRes `codec:"sigPayload" json:"sigPayload"`
+	EncryptionKey KID                    `codec:"encryptionKey" json:"encryptionKey"`
+	SigPayload    HelloRes               `codec:"sigPayload" json:"sigPayload"`
+	SdhBoxes      []SharedDhSecretKeyBox `codec:"sdhBoxes" json:"sdhBoxes"`
+}
+
+type SharedDhSecretKeyBox struct {
```

Comment:
So close! The only difference is that the `json` key of `ReceiverKID` has to be `receiver_kid` in order to talk to the server. But I don't know of a way to do that with avdl.